docs/development/git_workflow.rst

```diff
@@ -536,7 +536,24 @@ implement your work.
 After committing your changes you push your change to your repository
 `git push origin helping-with-PR116`.
 
- 
+
+Documentation Guidelines
```

Comment:
This should actually be a separate document and then added in toctree of `developer/index.rst`